### PR TITLE
cleanup: modify `Kubelet` to `kubelet`

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Modify `Kubelet` to `kubelet`

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/e2e/apps/daemon_restart.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ import (
 // 1. Daemons restart automatically within some sane time (10m).
 // 2. They don't take abnormal actions when restarted in the steady state.
 //	- Controller manager shouldn't overshoot replicas
-//	- Kubelet shouldn't restart containers
+//	- kubelet shouldn't restart containers
```
Comment (pravarag) [inline test/e2e/apps/daemon_restart.go:54]: Just wondering if this is the correct syntax for `kubelet` then we may need to update the same for `Controller manager` and `Scheduler` as well? or it'll be okay to use `Kubelet` as part of comments here?

Comment (my-git9) [reply test/e2e/apps/daemon_restart.go:54]: In my opinion, it doesn't matter whether you use `kubelet` or `Kubelet` on the annotation, it's just a handy modification here. Mainly think that the writing method of `kubelet` on the log output should be unified with other places to start with lowercase letters.

**Comment by bart0sh** [comment]
@my-git9 Thank you for your PR. Can you try to extend it to cover at least all places where "Kubelet" is used in the log messages.

Please, see [our Trivial Edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits) for more details.

**Comment by my-git9** [comment]
> @my-git9 Thank you for your PR. Can you try to extend it to cover at least all places where "Kubelet" is used in the log messages.
> 
> Please, see [our Trivial Edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits) for more details.

Updated

File: cmd/kubelet/app/server.go [hunk 2]
```diff
@@ -409,7 +409,7 @@ func UnsecuredDependencies(s *options.KubeletServer, featureGate featuregate.Fea
 // not be generated.
 func Run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Dependencies, featureGate featuregate.FeatureGate) error {
 	// To help debugging, immediately log version
-	klog.InfoS("Kubelet version", "kubeletVersion", version.Get())
+	klog.InfoS("kubelet version", "kubeletVersion", version.Get())
```
Comment (bart0sh) [inline cmd/kubelet/app/server.go:412]: This contradicts with the accepted logging style I guess. Log messages should be capitalized.

File: test/e2e/apps/daemon_restart.go [hunk 3]
```diff
@@ -319,7 +319,7 @@ var _ = SIGDescribe("DaemonRestart [Disruptive]", func() {
 		framework.ExpectNoError(e2erc.ScaleRC(ctx, f.ClientSet, f.ScalesGetter, ns, rcName, numPods+5, true))
 	})
 
-	ginkgo.It("Kubelet should not restart containers across restart", func(ctx context.Context) {
+	ginkgo.It("kubelet should not restart containers across restart", func(ctx context.Context) {
```
Comment (bart0sh) [inline test/e2e/apps/daemon_restart.go:322]: Here lowercased `kubelet` contradicts with other component names `Scheduler`, `Kube-proxy`, `Controller Manager` etc.

File: cmd/kubelet/app/server_windows.go [hunk 4]
```diff
@@ -37,7 +37,7 @@ func checkPermissions() error {
 
 	// For Windows user.UserName contains the login name and user.Name contains
 	// the user's display name - https://pkg.go.dev/os/user#User
-	klog.InfoS("Kubelet is running as", "login name", u.Username, "dispaly name", u.Name)
+	klog.InfoS("kubelet is running as", "login name", u.Username, "dispaly name", u.Name)
```
Comment (bart0sh) [inline cmd/kubelet/app/server_windows.go:41]: Sorry for being unclear in my previous comment. All **info** log messages should be capitalized.

File: pkg/kubelet/kubeletconfig/checkpoint/download.go [hunk 5]
```diff
@@ -191,12 +191,12 @@ func (r *remoteConfigMap) Download(client clientset.Interface, store cache.Store
 	}
 	// if we didn't find the ConfigMap in the in-memory store, download it from the API server
 	if cm == nil {
-		klog.InfoS("Kubelet config controller attempting to download remoteConfigMap", "apiPath", r.APIPath())
+		klog.InfoS("kubelet config controller attempting to download remoteConfigMap", "apiPath", r.APIPath())
```
Comment (bart0sh) [inline pkg/kubelet/kubeletconfig/checkpoint/download.go:194]: This one is still in lowercase.

File: pkg/kubelet/kubeletconfig/checkpoint/download.go [hunk 6]
```diff
@@ -191,12 +191,12 @@ func (r *remoteConfigMap) Download(client clientset.Interface, store cache.Store
...
 		cm, err = client.CoreV1().ConfigMaps(r.source.ConfigMap.Namespace).Get(context.TODO(), r.source.ConfigMap.Name, metav1.GetOptions{})
 		if err != nil {
 			return nil, status.DownloadError, fmt.Errorf("%s, error: %v", status.DownloadError, err)
 		}
-		klog.InfoS("Kubelet config controller successfully downloaded remoteConfigMap", "apiPath", r.APIPath(), "configMapUID", cm.UID, "resourceVersion", cm.ResourceVersion)
+		klog.InfoS("kubelet config controller successfully downloaded remoteConfigMap", "apiPath", r.APIPath(), "configMapUID", cm.UID, "resourceVersion", cm.ResourceVersion)
```
Comment (bart0sh) [inline pkg/kubelet/kubeletconfig/checkpoint/download.go:199]: The same here.

File: pkg/kubelet/kubeletconfig/status/status.go [hunk 7]
```diff
@@ -143,7 +143,7 @@ func (s *nodeConfigStatus) Sync(client clientset.Interface, nodeName string) {
 		return
 	}
 
-	klog.InfoS("Kubelet config controller updating Node.Status.Config")
+	klog.InfoS("kubelet config controller updating Node.Status.Config")
```
Comment (bart0sh) [inline pkg/kubelet/kubeletconfig/status/status.go:146]: and here

File: test/e2e_node/conformance/run_test.sh [hunk 8]
```diff
@@ -134,14 +134,14 @@ wait_kubelet() {
   local cur=1
   while [ $cur -le $maxRetry ]; do
     if curl -s $healthCheckURL > /dev/null; then
-      echo "Kubelet is ready"
+      echo "kubelet is ready"
```
Comment (bart0sh) [inline test/e2e_node/conformance/run_test.sh:137]: This one and below are also info messages, but I'm not sure if they should be capitalized or not. Can you look how it's done in other shell scripts?

Comment (my-git9) [reply test/e2e_node/conformance/run_test.sh:137]: I think `kubelet` is a fixed noun, and there is no problem with the whole lowercase of this noun

Comment (bart0sh) [reply test/e2e_node/conformance/run_test.sh:137]: That's one point of view. Another one is that info messages should be capitalized. You can see them capitalized in [the same file](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/conformance/run_test.sh#L119)

I'd suggest to keep them capitalized.

**Comment by bart0sh** [comment]
@my-git9 please, consider changing these lines as well:
```
cluster/gce/gci/configure-helper.sh:    echo "Must provide API server address to create Kubelet kubeconfig file!"
cluster/gce/gci/configure-helper.sh:    echo "Found Kubelet config file at ${KUBE_HOME}/kubelet-config.yaml"
cluster/gce/gci/configure.sh:  echo "Downloading Kubelet config file, if it exists"
cluster/gce/gci/flexvolume_node_setup.sh:echo "Restarting Kubelet..."
cluster/gce/gci/health-monitor.sh:      echo "Kubelet is unhealthy!"
cluster/gce/upgrade.sh:  echo "== $1 Node OS and Kubelet Versions =="
hack/make-rules/test-e2e-node.sh:  echo "Kubelet Config File: ${kubelet_config_file}"
hack/verify-generated-kubelet-plugin-registration.sh:echo "Generated Kubelet Plugin Registration api is up to date."
hack/verify-generated-kubelet-plugin-registration.sh:echo "Generated Kubelet Plugin Registration api is up to date."
```

File: pkg/kubelet/kubeletconfig/checkpoint/download.go [hunk 9]
```diff
@@ -182,7 +182,7 @@ func (r *remoteConfigMap) Download(client clientset.Interface, store cache.Store
 		cm, err = getConfigMapFromStore(store, r.source.ConfigMap.Namespace, r.source.ConfigMap.Name)
 		if err != nil {
 			// just log the error, we'll attempt a direct download instead
-			klog.ErrorS(err, "Kubelet config controller failed to check in-memory store for remoteConfigMap", "apiPath", r.APIPath())
+			klog.ErrorS(err, "kubelet config controller failed to check in-memory store for remoteConfigMap", "apiPath", r.APIPath())
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/kubeletconfig/checkpoint/download.go:185]: all output with `klog.ErrorS(err,` starts with upper case. It is a log statement, not the returned error that can be wrapped

**Review by SergeyKanzhelev — COMMENTED** [review]
/close

Some of this may be a good idea to change, but most changes are wrong - we use uppercase first letter in log statements

**Pull request closed without merging**